source/features/clone-branch.tsx

```diff
@@ -0,0 +1,107 @@
+import delegate, {DelegateEvent} from 'delegate-it';
+import gitBranch from 'octicon/git-branch.svg';
+import insertTextTextarea from 'insert-text-textarea';
+import React from 'dom-chef';
+import select from 'select-dom';
+import * as api from '../libs/api';
+import features from '../libs/features';
+import {getRepoURL, getRepoGQL} from '../libs/utils';
```

Comment:
Can you sort these imports?

Reply:
I thought this is how you want it? What is out of order? NPM then local ASC?

Reply:
"Tree" style, to almost-align them and make them easier to read

```js
import React from 'dom-chef';
import select from 'select-dom';
import gitBranch from 'octicon/git-branch.svg';
import insertTextTextarea from 'insert-text-textarea';
import delegate, {DelegateEvent} from 'delegate-it';
import * as api from '../libs/api';
import features from '../libs/features';
import {getRepoURL, getRepoGQL} from '../libs/utils';
```

Demo: The • follows the alignment


```js
import React from • 'dom-chef';
import select from • 'select-dom';
import gitBranch from • 'octicon/git-branch.svg';
import insertTextTextarea from • 'insert-text-textarea';
import delegate, {DelegateEvent} from • 'delegate-it';
import * as api from '../libs/api';
import features from '../libs/features';
import {getRepoURL, getRepoGQL} from '../libs/utils';
```